lib/peopleHR.ts

```diff
@@ -0,0 +1,235 @@
+import xml2js from 'xml2js';
+import { Window } from 'happy-dom';
+
+export interface JobPost {
+    title: string;
+    description: string;
+    link: string;
+    jobURL: string;
+    vacancyDescription: string;
+    department: string;
+    salaryRange: string;
+    city: string;
+    slug: string;
+}
+
+export interface JobSummary {
+    title: string;
+    description: string;
+    department: string;
+    city: string;
+    slug: string;
+}
+
+async function fetchPeopleHRFeed(): Promise<string> {
+    return fetch(`${process.env.PEOPLEHR_RSS_FEED_URL}`).then((response) =>
```

Comment:
**Suggestion**
> A fetch() promise only rejects when a network error is encountered (which is usually when there’s a permissions issue or similar). A fetch() promise does not reject on HTTP errors (404, etc.). Instead, a then() handler must check the Response.ok and/or Response.status properties. 

I think we'd want something like the snippet below, incase theres an issue fetching the resource we'd want to make it easier to debug.
```
            .then((response) => {
                if (response.ok) {
                    return response.text();
                }
                throw new Error(`Fetch PeopleHRFeed ${response.status}: ${response.statusText}`);
            })
```